scripts/openshift-helpers.sh

```diff
@@ -0,0 +1,181 @@
+	if [ "${prometheus_pod_running}" == "" ]; then
+		echo "Prometheus is not running, use 'scripts/prometheus_on_minikube.sh' to install."
+		exit 1
+	fi
+	echo "Prometheus is installed and running."
+}
+
+function openshift_first() {
+
+  #Create a namespace
+  echo "Create autotune namespace ${autotune_ns}"
+  kubectl create namespace ${autotune_ns}
+
+	kubectl_cmd="kubectl -n ${autotune_ns}"
+	echo "Info: One time setup - Create a service account to deploy autotune"
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_SA_MANIFEST}
+	check_err "Error: Failed to create service account and RBAC"
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_OPERATOR_CRD}
+	check_err "Error: Failed to create autotune CRD"
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_CONFIG_CRD}
+	check_err "Error: Failed to create autotuneconfig CRD"
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_QUERY_VARIABLE_CRD}
+	check_err "Error: Failed to create autotunequeryvariable CRD"
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_ROLE_MANIFEST}
+	check_err "Error: Failed to create role"
+
+	sed -e "s|{{ AUTOTUNE_NAMESPACE }}|${autotune_ns}|" ${AUTOTUNE_RB_MANIFEST_TEMPLATE} > ${AUTOTUNE_RB_MANIFEST}
+	${kubectl_cmd} apply -f ${AUTOTUNE_RB_MANIFEST}
+	check_err "Error: Failed to create role binding"
+
+	${kubectl_cmd} apply -f ${SERVICE_MONITOR_MANIFEST}
+	check_err "Error: Failed to create service monitor for Prometheus"
+}
+
+# You can deploy using kubectl
+function openshift_deploy() {
+	echo
+	echo "Creating environment variable in openshift cluster using configMap"
+	${kubectl_cmd} apply -f ${AUTOTUNE_CONFIGMAPS}/${cluster_type}-config.yaml
+
+	echo
+	echo "Deploying AutotuneConfig objects"
+	${kubectl_cmd} apply -f ${AUTOTUNE_CONFIGS}
+
+	echo
+	echo "Deploying AutotuneQueryVariable objects"
+	${kubectl_cmd} apply -f ${AUTOTUNE_QUERY_VARIABLES}/openshift-query-variable.yaml
+
+	echo "Info: Deploying autotune yaml to openshift cluster"
+
+	# Replace autotune docker image in deployment yaml
+	sed -e "s|{{ AUTOTUNE_IMAGE }}|${AUTOTUNE_DOCKER_IMAGE}|" ${AUTOTUNE_DEPLOY_MANIFEST_TEMPLATE} > ${AUTOTUNE_DEPLOY_MANIFEST}
+	sed -i "s|{{ OPTUNA_IMAGE }}|${OPTUNA_DOCKER_IMAGE}|" ${AUTOTUNE_DEPLOY_MANIFEST}
+
+	${kubectl_cmd} apply -f ${AUTOTUNE_DEPLOY_MANIFEST}
+	sleep 2
+	check_running autotune
+	if [ "${err}" != "0" ]; then
+		# Indicate deploy failed on error
+		exit 1
+	fi
+
+	# Get the Autotune application port in openshift
+	OPENSHIFT_IP=$(${kubectl_cmd} get pods -l=app=autotune -o wide -n monitoring -o=custom-columns=NODE:.spec.nodeName --no-headers)
+	AUTOTUNE_PORT=$(${kubectl_cmd} get svc autotune --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+	echo "Info: Access Autotune at http://$OPENSHIFT_IP:${AUTOTUNE_PORT}/listAutotuneTunables"
+	echo
+}
+
+
+function openshift_start() {
+	echo
+	echo "###   Installing autotune for openshift"
+	echo
+
+	# If autotune_ns was not set by the user
+	if [ -z "$autotune_ns" ]; then
+		autotune_ns="monitoring"
```

Comment:
Let us set the default for openshift to be `openshift-tuning`. Please create a global variable and assign it this value and use that variable everywhere